### ISSUE TITLE
NTR; pulmo: pulmonary Necrosis

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6451]
**Preferred term label:**
pulmonary necrosis

**Synonyms**
unprogrammed cellular death

**Definition (free text, please give PubMed ID)**
Cellular death affecting one or more parts of the lung. Necrosis is caused by divers external and internal factors which result in the unregulated digestion of cell components, in contrast to apoptosis the programmed cellular death.

Comment: Cell blebbing (bulky morphology of the cell, disruptin of cell membranes and organelles) and nucleus changes (shrunken nucleus fragments and chromatin condesation) is visible with electron microscope and DAPI staining. The release of cell contents initiates an inflammatory response in the surrounding tissue is detectable with H&E staining. 

**Parent term (use hpo.jax.org/app)**
Interstitial pulmonary abnormality HP:0006530

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
lmu: chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
added
